resources/home/dnanexus/run_workflows/tests/test_manage_dict.py

```
@@ -598,88 +599,94 @@
+    def test_adding_project_id(self, other_inputs):
         """
         Test for finding INPUT-dx_project_id and replacing with project_id
         from args Namespace object
         """
         assert (
-            self.output["my_project_id"] == "project-12345"
+            other_inputs["my_project_id"] == "project-12345"
         ), "INPUT-dx_project_id not correctly replaced"
 
-    def test_adding_project_name(self):
+    def test_adding_project_name(self, other_inputs):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_adding_project_name`

Add return type annotation: `None`

(ANN201)